__tests__/api.test.js

```diff
@@ -0,0 +1,60 @@
+const request = require('supertest')
+const seed = require('../db/seeds/seed');
+
+const topicData = require('../db/data/test-data/topics');
+const articleData = require('../db/data/test-data/articles');
+const userData = require('../db/data/test-data/users');
+const commentData = require('../db/data/test-data/comments');
```

Comment:
Fun Fact: We set up the index file on purpose so you can just require in the index file from test-data, so you can then just invoke seed with the variable you used to require it in!
e.g.:
`const data = require('../db/data/test-data/index');`
and then
`return seed(data)`